doc/api/assert.markdown

```diff
@@ -3,9 +3,10 @@
     Stability: 3 - Locked
 
 The `assert` module provides a simple set of assertion tests that can be used
-to test invariants and implement unit tests. While the `assert` module is
-generally intended for internal use by Node.js itself, it can be used by user
-code calling `require('assert')`.
+to test invariants and implement unit tests. It is recommended that users of
+Node.js choose a third party assertion library from [npm][] instead of relying
+on `assert`.  That being said, `assert` can be used by user code calling
```

Comment:
OK, I see, searching for assert on npmjs.org gets you pages of results, none of which are useful to replace assert for invariants, and none of which are a unit test framework (which is probably what the reader wanted).

The usage of `assert` we want to warn people away from is using it as a unit testing library, such as mocha, tap, and nunit, and none of those are even in the first page of search results when using npmjs.org search.

Of course, I can't find a search term (other than "mocha") that yields mocha in the first page. :-( This is likely because npmjs.org search is unranked, so not so helpful, googling for node test framework works better.